gem/styles/gem-rwanda/modules/_m-profiles.scss

```diff
@@ -14,6 +14,11 @@
       margin-top: 0px;
       color: $color-pink;
     }
+    & .heading--subtitle {
```

Comment:
@KaitCrawford I think you can remove the &, it points to the parent selector if child selector is nested.

Reply:
oh! I thought it was just a placeholder, string concatenation thing :joy: